templates/interfaces.j2

```diff
@@ -1,49 +1,127 @@
 # vim: sw=4 ts=4 ft=jinja
 # {{ ansible_managed }}
 
+
+{% if network_management_pre_up is defined %}
+post_up {{ network_management_pre_up }}
+{% endif %}
+
 auto lo
 iface lo inet loopback
 
-{% for interface in bridges %}
+{% if network_management_clear_bridges %}
+# Delete all network bridges
+pre-up for br in `ovs-vsctl list-br`; do ovs-vsctl --if-exists "$br"; done
+{% endif %}
+
+################################################################################
+# Configure all network interfaces                                             #
+################################################################################
+{% for interface in interfaces %}
+{% if interface.bring_up|default(network_management_default_bring_up) %}
+auto {{ interface.name }}
+{% endif -%}
+
+iface {{ interface.name }} inet {{ interface.type|default(network_management_default_type) }}
+{% if interface.type|default(network_management_default_type) == "dhcp" %}
+	hostname {{ interface.hostname|default(ansible_hostname) }}
+	{% if interface.leasetime is defined %}leasetime {{ interface.leasetime }}{% endif %}
+	{% if interface.metric is defined %}metric {{ interface.metric }}{% endif %}
+{% elif interface.type|default(network_management_default_type) == "static" -%}
+
+{% for ip in interface.ips %}
+	{% if not loop.first %}
```

Comment:
Is the first entry deterministically the loopback interface? Is there no sorting possible in-between?

Reply:
?